### PR TITLE
Fix [ pairtypes ] and [ nonbond_params ]

### PR DESCRIPTION
**Pull request by leeping**
With this change, the DPPC test case now works.

### CODE REVIEW

**Comment by leeping** [comment]
https://github.com/ParmEd/ParmEd/pull/152

**Comment by leeping** [comment]
https://github.com/ParmEd/ParmEd/pull/152

**Comment by leeping** [comment]
The latest commit removes several duplicate files and the `12A` and `12B` test cases.  These shouldn't be needed anymore now that the full DPPC test case is working.

File: chemistry/gromacs/gromacstop.py [hunk 1]
```diff
@@ -622,10 +622,10 @@ def read(self, fname, defines=None, parametrize=True):
...
                     sig *= 10 # Convert to Angstroms
                     eps *= u.kilojoule.conversion_factor_to(u.kilocalorie)
-                    params.nbfix_types[(a1, a2)] = (eps, sig*2**(-1/6))
-                    params.nbfix_types[(a2, a1)] = (eps, sig*2**(-1/6))
-                    params.atom_types[a1].add_nbfix(a2, sig*2**(-1/6), eps)
-                    params.atom_types[a2].add_nbfix(a1, sig*2**(-1/6), eps)
+                    params.nbfix_types[(a1, a2)] = (eps, sig*2**(1/6))
+                    params.nbfix_types[(a2, a1)] = (eps, sig*2**(1/6))
+                    params.atom_types[a1].add_nbfix(a2, sig*2**(1/6), eps)
+                    params.atom_types[a2].add_nbfix(a1, sig*2**(1/6), eps)
```
Comment (swails) [inline chemistry/gromacs/gromacstop.py]: Oh man, that was stupid.  Thanks.

File: chemistry/gromacs/gromacstop.py [hunk 2]
```diff
@@ -830,6 +854,16 @@ def update_typelist_from(ptypes, types):
...
+        if len(true_14 - gmx_pair) > 0:
+            zero_pairtype = NonbondedExceptionType(0.0, 0.0, 0.0, list=self.adjust_types)
+            self.adjust_types.append(zero_pairtype)
+            num_zero_14 = 0
+            for zero_14 in (true_14 - gmx_pair):
+                self.adjusts.append(NonbondedException(zero_14[0], zero_14[1], zero_pairtype))
+                num_zero_14 += 1
+            warnings.warn('%i 1-4 pairs were missing from the [ pairs ] section and were set to zero; make sure you know what you\'re doing!' % num_zero_14)
+        if len(gmx_pair - true_14) > 0:
+            warnings.warn('The [ pairs ] section contains %i exceptions that aren\'t 1-4 pairs; make sure you know what you\'re doing!' % (len(gmx_pair - true_14)))
```
Comment (swails) [inline chemistry/gromacs/gromacstop.py]: Can you turn these into `GromacsTopologyWarning`s?

```Python
warnings.warn('The [ pairs ] section contains ...', GromacsTopologyWarning)
```

**Comment by swails** [comment]
Actually I'll go ahead and make those changes.  It's ~5:30 PDT right now, and I want to get the Amber part fixed :).

**Pull request merged**